packages/calendar/src/index.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react'
+import {useState} from 'react'
+import getUserLocale from 'get-user-locale'
+import {useDefaults} from '@smashing/theme'
+import {CalendarProps, CalendarAppearanceType} from './types'
+import {StyledContainer, StyledInput, StyledCalendar} from './styles'
+
+const Calendar: React.FC<CalendarProps> = ({
+  onChange = (date: Date) => {},
+  value,
+  ...props
+}) => {
+  const [open, setOpen] = useState(false)
+  const defaults = useDefaults('calendar', props, {
+    appearance: 'default' as CalendarAppearanceType
+  })
+
+  const getTwoDigitNumber = variable => {
+    return variable > 9 ? variable : '0' + variable
+  }
+
+  const getFormattedDate = () => {
+    if (!value) {
+      return ''
+    }
+    const day = value.getDate()
+    const month = value.getMonth() + 1
+    const year = value.getFullYear()
+
+    return `${getTwoDigitNumber(day)}/${getTwoDigitNumber(month)}/${year}`
+  }
+
+  const onDateChange = chosenDate => {
+    setOpen(false)
+    onChange(chosenDate)
+  }
+
+  const formatShortWeekday = (locale, formattedDate) => {
+    return formattedDate
+      .toLocaleString(locale || getUserLocale(), {
+        weekday: 'short'
+      })
+      .slice(0, 2)
+  }
+
+  return (
+    <StyledContainer>
+      <StyledInput
+        // TODO add appearance for input
+        open={open}
+        placeholder="DD/MM/YY"
+        readOnly
+        onClick={() => setOpen(!open)}
```

Comment:
```suggestion
        onFocus={() => setOpen(true)}
        onBlur={() => setOpen(false)}
```

Reply:
@Idered It isn't working because when user clicks something in calendar (ie. next month arrow) input loses focus and calendar disappear.

Reply:
Right, but onFocus should be good. Instead of onBlur you should add event listener to document which listen for clicks outside of input and calendar popup.

Simple onClick to toggle won't work here

Reply:
Done